cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -116,11 +108,12 @@ func NewEthereumSub(inBuf io.Reader, rpcURL string, cdc *codec.Codec, validatorM
 }
 
 // LoadTendermintCLIContext : loads CLI context for tendermint txs
-func LoadTendermintCLIContext(appCodec *amino.Codec, validatorAddress sdk.ValAddress, validatorName string,
-	rpcURL string, chainID string, sugaredLogger *zap.SugaredLogger) (sdkContext.CLIContext, error) {
+func LoadTendermintCLIContext(validatorAddress sdk.ValAddress, validatorName string,
+	rpcURL string, chainID string, sugaredLogger *zap.SugaredLogger) (client.Context, error) {
 	// Create the new CLI context
-	cliCtx := sdkContext.NewCLIContext().
-		WithCodec(appCodec).
+	encodingConfig := sifapp.MakeTestEncodingConfig()
```

Comment:
We shouldn't be calling MakeTestEncodingConfig here:

// MakeTestEncodingConfig creates an EncodingConfig for testing.
// This function should be used only internally (in the SDK).
// App user should'nt create new codecs - use the app.AppCodec instead.
// [DEPRECATED]